tools/update-webpage.sh

```diff
@@ -41,6 +43,8 @@ titles[$PORT_API_MD]="Port API"
 titles[$REFERENCE_COUNTING_MD]="Reference Counting"
 titles[$DEBUGGER_MD]="Debugger"
 titles[$CODING_STANDARDS_MD]="Coding Standards"
+titles[$EXT_REFERENCE_ARG_MD]="External Argument Reference"
+titles[$EXT_REFERENCE_HANDLER_MD]="External Handler Reference"
```

Comment:
I've omitted the 'Reference' in order to reduce the width of the options and the titles with `:` are enclosed with quote-marks to avoid interpret them as a key-value pair.